src/ngraph/op/subtract.hpp

```diff
@@ -50,9 +50,43 @@ namespace ngraph
                 virtual void generate_adjoints(autodiff::Adjoints& adjoints,
                                                const NodeVector& deltas) override;
             };
-        }
+        } // namespace v0
+
+        namespace v1
+        {
+            /// \brief Elementwise subtraction operation.
+            class Subtract : public util::BinaryElementwiseArithmetic
+            {
+            public:
+                NGRAPH_API
```

Comment:
`NGRAPH_API` macro for `type_info` should be removed

Reply:
Done